eiffel-vocabulary/EiffelServiceDeployedEvent.md

```diff
@@ -22,17 +22,17 @@ It implies that the service instance is ready to be started.
 ## Data Members
```

Comment:
In the first paragraph "EiffelServiceDeployedEvent" I noticed a spelling mistake, which could also be fixed with this PR :)

Reply:
Hi, Okay will fix in this PR